hazelcast/src/main/java/com/hazelcast/internal/monitor/impl/LocalReplicatedMapStatsImpl.java

```diff
@@ -400,14 +405,22 @@ public void fromJson(JsonObject json) {
         ownedEntryCount = getLong(json, "ownedEntryCount", -1L);
         ownedEntryMemoryCost = getLong(json, "ownedEntryMemoryCost", -1L);
         creationTime = getLong(json, "creationTime", -1L);
-        totalGetLatencies = getLong(json, "totalGetLatencies", -1L);
-        totalPutLatencies = getLong(json, "totalPutLatencies", -1L);
-        totalRemoveLatencies = getLong(json, "totalRemoveLatencies", -1L);
+        totalGetLatenciesNanos = convertMillisToNanos(getLong(json, "totalGetLatencies", -1L));
+        totalPutLatenciesNanos = convertMillisToNanos(getLong(json, "totalPutLatencies", -1L));
+        totalRemoveLatenciesNanos = convertMillisToNanos(getLong(json, "totalRemoveLatencies", -1L));
         maxGetLatency = getLong(json, "maxGetLatency", -1L);
         maxPutLatency = getLong(json, "maxPutLatency", -1L);
         maxRemoveLatency = getLong(json, "maxRemoveLatency", -1L);
     }
 
+    private static long convertNanosToMillis(long nanos) {
```

Comment:
nit: extract this and the following one methods into a common module (say, `TimeUtil`) and reuse them here and in `LocalMapStats`.